packages/material/scss/upload/_theme.scss

```diff
@@ -38,18 +37,14 @@ $upload-success-text: $text-color !default;
             color: $upload-disabled-text;
 
             &:hover {
-                background: white;
+                background-color: transparent;
```

Comment:
Don't we have convetsion to use only `background` property rule for color? :)

Reply:
Not necessarily. Setting the `background` property sets _a lot_ of other properties, and we try to use it sparingly. It seems that in this case `background-color` suffices, thus the change :)